po/cs.po

```diff
@@ -116,21 +116,23 @@ msgid "Grant Access"
 msgstr "Schválit přístup"
 
 #: src/location.c:535
-msgid "Grant Access to Your Location?"
+#, fuzzy
+msgid "Allow Access to Your Location?"
 msgstr "Schválit přístup k vaší poloze?"
 
 #: src/location.c:536
-msgid "An application wants to use your location."
+#, fuzzy
+msgid "An application wants to known the location of this device."
 msgstr "Nějaká aplikace chce použít vaši polohu."
 
 #: src/location.c:548
-#, c-format
-msgid "Give %s Access to Your Location?"
+#, fuzzy, c-format
+msgid "Allow %s to Access to Your Location?"
```

Comment:
Nope, the 'to' makes it incorrect English.

Reply:
Note that that's the second "to" that isn't needed, not the first one!

For example, "Allow Maps to Access Your Location".